hack/tools/go.mod

```diff
@@ -7,20 +7,21 @@ require (
 	github.com/cespare/prettybench v0.0.0-20150116022406-03b8cfe5406c
 	github.com/client9/misspell v0.3.4
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.45.0
+	github.com/golangci/golangci-lint v1.46.2
```

Comment:
@liggitt is this a file that you use to review for code-organization/dependency management?